cmake/get_cpm.cmake

```diff
@@ -1,3 +1,7 @@
+# SPDX-License-Identifier: MIT
+#
```

Comment:
This empty line is needed to prevent `cmake-format` from automatically line-wrapping these comments.